### PR TITLE
Review workflow UI improvements.

### LINKED ISSUE #317: [Signoff] Confusing screen after having approved changes
This is what you get after you've approved changes as a reviewer:

screenshot at 16 44 42

This is rather confusing! /cc @leplatrem

Also, the notification message is confusing as well, it should probably say *Changes approved, collection signature has been requested.*

### PR DESCRIPTION
**Pull request by n1k0**
This fixes #317 and other issues. WiP.

### CODE REVIEW

**Comment by n1k0** [comment]
r=? @leplatrem

**Comment by leplatrem** [comment]
I think you forgot to add the tests file. Good news is that CI is not gonna break :)

**Comment by n1k0** [comment]
Ah sorry for that, pushed in https://github.com/Kinto/kinto-admin/pull/319/commits/b4aadb37414d03714491452a72580d3c95001ea0

File: test/plugins/signoff/sagas_test.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+  describe("handleRequestReview()", () => {
+    let bucket, collection, getState, handleRequestReview;
+
+    before(() => {
+      collection = {
+        getData() {},
+        setData() {},
+      };
+      bucket = {collection() {return collection;}};
+      setClient({bucket() {return bucket;}});
```
Comment (leplatrem) [inline test/plugins/signoff/sagas_test.js]: nit: maybe put all that in a `fakeClient` variable shared with all tests?

File: src/plugins/signoff/components.js [hunk 2]
```diff
@@ -19,6 +19,44 @@ import AdminLink from "../../components/AdminLink";
...
+  if (!user.id) {
+    return false;
+  }
+  const {signer={}} = capabilities;
+  const {[groupkey]: defaultGroupName} = signer;
+  const {[groupkey]: groupName=defaultGroupName} = source;
+  const {id: userId} = user;
+  console.log(bucketState);
+  const {groups} = bucketState;
+  const editorGroup = groups.find(g => g.id === groupName);
```
Comment (leplatrem) [inline src/plugins/signoff/components.js]: nit: `editorGroup` →  `group`

File: src/plugins/signoff/components.js [hunk 3]
```diff
@@ -19,6 +19,44 @@ import AdminLink from "../../components/AdminLink";
 import { ProgressBar, ProgressStep } from "./ProgressBar.js";
 
 
+function isMember(groupkey, source, sessionState, bucketState) {
+  const {serverInfo: {user={}, capabilities}} = sessionState;
+  if (!user.id) {
+    return false;
+  }
+  const {signer={}} = capabilities;
+  const {[groupkey]: defaultGroupName} = signer;
```
Comment (leplatrem) [inline src/plugins/signoff/components.js]: We could even do this in one line without intermediary `capabilities` and `signer` variables :) `#PerlFTW`

File: src/plugins/signoff/components.js [hunk 4]
```diff
@@ -19,6 +19,44 @@ import AdminLink from "../../components/AdminLink";
...
+  const {[groupkey]: groupName=defaultGroupName} = source;
+  const {id: userId} = user;
+  console.log(bucketState);
```
Comment (leplatrem) [inline src/plugins/signoff/components.js]: leftover

File: src/plugins/signoff/components.js (hunk 3 above)
Comment (n1k0) [inline src/plugins/signoff/components.js]: You mean

```js
const {serverInfo: {capabilities: {signer: {[groupKey]: defaultGroupName}={}}}} = sessionState;
```

LOLNOPE 😬

**Comment by leplatrem** [comment]
:shipit:

**Pull request merged**